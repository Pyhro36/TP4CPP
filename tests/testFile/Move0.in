PC convex0 5 6 3 6 3 -4 5 -4
S seg0 0 4 10 0
S seg1 0 0 10 4
R rec0 0 4 10 0
PC pc 0 4 10 0 0 0
OI oi seg0 seg1 rec0 pc
S seg2 0 4 10 0
R rec -10 -5 1 5
OR or seg2 rec
LIST
MOVE convex0 15 18
MOVE seg0 -5 -4
MOVE rec0 -7 8
MOVE pc 10 0
MOVE oi 25 -48
MOVE or 0 2
LIST
EXIT
