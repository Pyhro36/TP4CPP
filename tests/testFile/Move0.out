Shape handler 1.0 :
OK
OK
OK
OK
OK
OK
OK
OK
OK
PC convex0 5 6 3 6 3 -4 5 -4
S seg0-oi 0 4 10 0
S seg1-oi 0 0 10 4
R rec0-oi 0 4 10 0
PC pc-oi 0 4 10 0 0 0
OI oi seg0-oi seg1-oi rec0-oi pc-oi
DELETE seg0-oi seg1-oi rec0-oi pc-oi
S seg2-or 0 4 10 0
R rec-or -10 -5 1 5
OR or seg2-or rec-or
DELETE seg2-or rec-or
PC pc 0 4 10 0 0 0
R rec -10 -5 1 5
R rec0 0 4 10 0
S seg0 0 4 10 0
S seg1 0 0 10 4
S seg2 0 4 10 0

OK
OK
OK
OK
OK
OK
PC convex0 20 24 18 24 18 14 20 14
S seg0-oi 25 -44 35 -48
S seg1-oi 25 -48 35 -44
R rec0-oi 25 -44 35 -48
PC pc-oi 25 -44 35 -48 25 -48
OI oi seg0-oi seg1-oi rec0-oi pc-oi
DELETE seg0-oi seg1-oi rec0-oi pc-oi
S seg2-or 0 6 10 2
R rec-or -10 -3 1 7
OR or seg2-or rec-or
DELETE seg2-or rec-or
PC pc 10 4 20 0 10 0
R rec -10 -5 1 5
R rec0 -7 12 3 8
S seg0 -5 0 5 -4
S seg1 0 0 10 4
S seg2 0 4 10 0

